docs/oid-info.md

```diff
@@ -23,35 +23,49 @@ The `.1` is added to the root OID for sigstore for all OIDs set by Fulcio.
 This contains the `issuer` claim from the OIDC Identity Token that was
 presented at the time the code signing certificate was requested to be created.
 This claim is the URI of the OIDC Identity Provider that digitally signed the
-identity token.
+identity token. For example: https://token.actions.githubusercontent.com.
 
-### 1.3.6.1.4.1.57264.1.2 | GitHub Workflow Trigger
+### 1.3.6.1.4.1.57264.3.1 | Source URI
```

Comment:
The OID prefix `1.3.6.1.4.1.57264.1` is specifically registered for Fulcio so I think its important that these new definitions all use that prefix instead of changing to the `1.3.6.1.4.1.57264.3` prefix. Ref: http://oid-info.com/get/1.3.6.1.4.1.57264.1

Reply:
Ah TIL! I agree that changing the prefix seems unnecessary and can lead to confusion though.

Reply:
Yeah I can see how changing the prefix is confusing unless we change all of them. Can the suffix increment to double digits? e.g. `1.3.6.1.4.1.57264.1.12`, assuming we want to keep all existing extensions as they and add the new ones following `1.3.6.1.4.1.57264.1.7 | OtherName SAN`. So "Source URI" would end up as `1.3.6.1.4.1.57264.1.8` etc.

Reply:
yep, the suffix can increase to two digits. I think that would be fine to start from `.8`.